utils/docker-ce/Makefile

```diff
@@ -30,8 +30,9 @@ define Package/docker-ce
   CATEGORY:=Utilities
   TITLE:=Docker Community Edition
   URL:=https://www.docker.com/
-  DEPENDS:=$(GO_ARCH_DEPENDS) @TARGET_x86_64 +btrfs-progs +libdevmapper +containerd +cgroupfs-mount +ca-certificates +iptables-mod-extra +DOCKER_SECCOMP:libseccomp \
-           +kmod-ikconfig +kmod-veth +kmod-br-netfilter +kmod-nf-ipvs
+  DEPENDS:=$(GO_ARCH_DEPENDS) @TARGET_x86_64 +containerd +tini +libnetwork +cgroupfs-mount +btrfs-progs +libdevmapper +ca-certificates \
+           +iptables-mod-extra +DOCKER_SECCOMP:libseccomp +kmod-ikconfig +kmod-veth +kmod-br-netfilter +kmod-nf-ipvs
```

Comment:
Feel like grouping by type then sorting alphabetically?  It could help review

Reply:
I wasn't sure how you meant, is what I've done now acceptable?

Reply:
Much better.  Now that GitHub can display with word-diff.  Thank you.